XamarinCommunityToolkitSample.UWP/Xamarin.CommunityToolkit.Sample.UWP.csproj

```diff
@@ -143,7 +143,7 @@
     </Page>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Forms" Version="4.6.0.1141" />
+    <PackageReference Include="Xamarin.Forms" Version="4.8.0.1269" />
```

Comment:
Again, is this necessary? Although I don't mind using a newer version for the sample app, but can you then also upgrade iOS and Android?

Reply:
Ios and Android were already bumped...check main.

I had to bump uwp to match because the sample app used the new online grid row/column Def and the converter isn't in 4.6

Reply:
In that case, no problem!

Reply:
it's those dang Mac users, not updating UWP 😄

Reply:
Guilty...